hooks/useLocalStorage.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+
+export const useLocalStorage = (key: string, initialValue?: any) => {
```

Comment:
As mentioned, we can use generic here
```suggestion
export function useLocalStorage<T>(key: string, initialValue?: T) {
```